grails-app/views/location/edit.gsp

```diff
@@ -17,10 +17,7 @@
 
 <body>
 <div class="body">
-
-    <g:if test="${flash.message}">
-        <div class="message">${flash.message}</div>
-    </g:if>
+    <g:renderMessage />
```

Comment:
I like this solution since rendering messages with HTML tags doesn't work well in Grails 1. The first situation that came to my mind was the message after logging in with incorrect credentials. (This message works on Grails 3 - the same code, without any changes) 👍👍